src/find_topic_tokens.py

```diff
@@ -0,0 +1,106 @@
+import os
+import gin
+import numpy as np
+import torch
+from src.utils.model import LM
+from scipy.spatial.distance import euclidean
+import csv
+from src.utils.dataset import Cord19
+
+@gin.configurable()
+def find_topic_tokens(run_name, dim, hold_out_file='./.data/countries_test.txt', sigma=0.8):
+
+    token_hidden_states, token_hidden_states_labels = torch.load('./models/{}_token_hidden_state.pt'.format(run_name))
+
+    token_hidden_states_labels_lower = [l.lower() for l in token_hidden_states_labels]
+    uniq_labels = set(token_hidden_states_labels_lower)
+    hiddens_state_means = {}
+    for lbl in uniq_labels:
+        indicies = np.where(np.array(token_hidden_states_labels_lower) == lbl)
+        mean = np.mean(token_hidden_states[indicies], axis=0).reshape(1, -1)
```

Comment:
the mean of all hidden states for a particular token right?

Reply:
correct